practice-one/src/Components/Common/horizontalCard.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from './Card';
+
+const HorizontalCard = ({
+  headerText = '',
+  title = '',
+  date = '',
+  author = '',
+}) => (
+  <Card variant="primary" size="medium">
+    <div className="flex justify-between items-end border-t-2 border-t-gray-500">
+      <div className="card-horizontal-left pt-5">
+        <Card.CardContent>
+          <h3 className="text-xl font-bold mb-6">{title}</h3>
+        </Card.CardContent>
+        <Card.CardFooter>
+          <h4>
+            {date}
+            ,

```

Comment:
use HTML char code

Reply:
Done,  used {String.fromCharCode(44)} for `,`